modules/initrd.nix

```diff
@@ -1,12 +1,12 @@
 { config, pkgs, lib, utils, ... }:
 
 let
+  udev = pkgs.systemdMinimal;
```

Comment:
Worth drawing your attention to this, as it's core code not port code. I don't *imagine* that using systemdMinimal for udev in the initrd should break any other device, but that might be just my lack of imagination

Reply:
I don't think so either since we're cherry-picking the actual udev description files.